htdocs/static/js/src/netmap/control_view.js

```diff
@@ -147,20 +147,6 @@ define([
             Backbone.EventBroker.trigger('netmap:fixNodes');
         },
 
-        fireToggleForce: function (e) {
```

Comment:
Good point. In short, we mostly use force physics to do the initial redistribution of the drawn nodes to provide sensible spacing out-of-the-box. After this has been done, there is really no need to have force physics acting on the nodes, as it only provides overhead, sometimes a little too much when dealing with large topologies. Should probably add a small note about this mechanism in the main view.